certbot/compat/os.py

```diff
@@ -29,3 +29,15 @@
 
 # Clean all remaining importables that are not from the core os module.
 del ourselves, std_os, std_sys
+
+
+# The os.open function on Windows will have the same effect than a bare os.chown towards the given
+# mode, and will create a file with the same flaws that what have been described for os.chown.
+# So upon file creation, security.take_ownership will be called to ensure current user is the owner
+# of the file, and security.chmod will do the same thing than for the modified os.chown.
+# Internally, take_ownership will update the existing metdata of the file, to set the current
```

Comment:
Internally to what?

Should all of this documentation be here? This function is just raising an error. Maybe it should say what to call instead, and have that documentation there?

Reply:
Indeed. I simplified the comment here.